src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -34,10 +34,19 @@ public struct PlayerBufferingTime
         /// Initializes a new instance of the PlayerBufferingTime struct.
         /// </summary>
         /// <param name="preBufferMillisecond">A duration of buffering data that must be prerolled to start playback.</param>
-        /// <param name="reBufferMillisecond">A duration of buffering data that must be prerolled to resume playback
-        /// if player is paused for buffering internally.</param>
+        /// Except 0 and -1, setting at least 1000 milliseconds is recommended to ensure the normal buffering operation.
+        /// 0 : use platform default value which could be different depending on the streaming type and network status. (the initial value)
```

Comment:
The current sentence is not bad. 
How about the following sentence?
```
// 0 : use platform default value, which depends on the streaming type and network status. It is set as the initial value of this parameter.
```

It depend on you to change the sentence.